README.md

```diff
@@ -111,7 +113,11 @@ Please refer to the details on how the ngx-prefetch works [here](docs/HOW_IT_WOR
         "crossorigin": true,
         "production": false,
         "staticsFullPath": "https://my-web-app.com/path/to/my-app/",
-        "localizationPattern": "/localizations/${language}.json"
+        "localizationPattern": "/localizations/${language}.json",
+        "fallbackLocalesMap": {
+            'en-CA': 'en-GB',
+            'en-US': 'en-GB'
```

Comment:
it could be nice to support wildcards/regex to be able to do stuff like
```
"fallbackLocalesMap": {
     'en-*': 'en-GB',
     '*-FR': 'fr-FR'
}
```